src/main/steps/applicant1/confirm-your-joint-application/content.ts

```diff
@@ -91,8 +92,25 @@ const en = ({ isDivorce, partner, userCase, userEmail, isApplicant2 }: CommonCon
   subHeading5: `Reason for  ${isDivorce ? 'the divorce' : 'ending the civil partnership'}`,
   line18: `The ${isDivorce ? 'marriage' : 'relationship'} has broken down irretrievably (it cannot be saved).`,
   subHeading6: 'Financial order application',
-  applicant1FinancialOrderYes: 'Applicant 1 is applying to the court for financial orders',
-  applicant2FinancialOrderYes: 'Applicant 2 is applying to the court for financial orders',
+  applicant1FinancialOrderYes: `Applicant 1 is intending to apply to the court for financial orders ${
+    userCase.whoIsFinancialOrderFor?.includes(FinancialOrderFor.APPLICANT) ? 'for the applicant' : ''
+  }${
+    isEqual(userCase.whoIsFinancialOrderFor, [FinancialOrderFor.APPLICANT, FinancialOrderFor.CHILDREN]) ? ' and ' : ''
+  }${
+    userCase.whoIsFinancialOrderFor?.includes(FinancialOrderFor.CHILDREN)
+      ? 'for the children of the applicant and the respondent.'
+      : '.'
+  }`,
+  applicant2FinancialOrderYes: `Applicant 2 is applying to the court for financial orders ${
+    userCase.applicant2WhoIsFinancialOrderFor?.includes(FinancialOrderFor.APPLICANT) ? 'for the applicant' : ''
+  }${
+    isEqual(userCase.applicant2WhoIsFinancialOrderFor, [FinancialOrderFor.APPLICANT, FinancialOrderFor.CHILDREN])
```

Comment:
Can we guarantee that [FinancialOrderFor.APPLICANT, FinancialOrderFor.CHILDREN] will always be in the same order? cos isEqual will return false for [FinancialOrderFor.CHILDREN, FinancialOrderFor.APPLICANT]

Reply:
for some reason using sort broke the comparison before. tho now im using lodash so it might work and ill add it in.